### PR TITLE
implement fix for #issue6

### PR DESCRIPTION
**Pull request by gbeane**
Fix for issue #6

### CODE REVIEW

File: src/feature_extraction/feature_group_base_class.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def window(self, identity: int, window_size: int,
         feature_modules = self._init_feature_mods(identity)
         return {
             name: mod.window(identity, window_size, per_frame_values[name]) for name, mod in
-            feature_modules.items()
+            feature_modules.items() if name in per_frame_values.keys()
```
Comment (gbeane) [inline src/feature_extraction/feature_group_base_class.py:56]: this line is the main fix

**Comment by gerring** [comment]
It would appear I am not on the project and cannot review merge requests?

**Comment by gerring** [comment]
Change looks okay but have little idea what it does or if it breaks or fixes things because lack of tests. I assume it is fine.

**Comment by SkepticRaven** [comment]
I tested on one of my pose_v2 projects, and seems to re-calculate window features.
We'll see what Leinani's tests return, since it may be related to a specific combination of pose file versions available.

**Review by SkepticRaven — APPROVED** [review]

**Comment by gbeane** [comment]
found one more thing I need to troubleshoot related to Leinani's issue

**Pull request closed without merging**